lib/index.js

```diff
@@ -1,8 +1,7 @@
-/*!
- * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
+/*
+ * Copyright (c) 2019-2020 Digital Bazaar, Inc. All rights reserved.
  */
-'use strict';
 
-// translate `main.js` to CommonJS
-require = require('esm')(module);
-module.exports = require('./main.js');
+import LocalVerifiableCredentialStore from './LocalVerifiableCredentialStore';
+
+export default LocalVerifiableCredentialStore;
```

Comment:
I think we would want to have named exports here.  It's almost always the case that more things will be exported here in the fullness of time.  If we don't set things up properly now, that will be a breaking change later.

Not sure off the top if this is the right syntax.
```suggestion
export default {LocalVerifiableCredentialStore};
```